### DISCUSSION TITLE
How to silent install development headers and libraries for C and C++?

### DISCUSSION THREAD

**Discussion by fitmap** [ImageMagick/ImageMagick#4869] [General]
ImageMagick-7.1.0-25-Q16-HDRI-x64-dll.exe /SP- /silent /dir=C:/ImageMagick /COMPONENTS=?

**Comment by pprasanthk** [comment]
Hi, 

I used this command: 
ImageMagick-7.1.1-21-Q8-x86-dll.exe /SP /VERYSILENT /TASKS="install_devel" /DIR="C:\TEMP\ImageMagick-Q8-x86\"

--
Thanks
Prasanth
